.gitlab/deception.gitlab-ci.yml

```diff
@@ -0,0 +1,74 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
+
+hykkt-test-deception:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "a100_shared"
+    MY_CLUSTER: "deception"
+  extends:
+    - .pnnl_tags_template
+    - .hykkt_script_template  
+
+.report-status:
+  image: mrnonz/alpine-git-curl:alpine3.16
+  variables:
+    GIT_STRATEGY: none
+    STATUS_PROJECT: ORNL/hykkt
+    STATUS_NAME: CentOS-7-X86_64-gcc/9.1.0-cuda-11.4
+  extends:
+  - .pnnl_nonhpc_tags
+  script:
+    - |
+      set -x
+      curl -L \
+      -X POST \
+      -H @${GITHUB_CURL_HEADER}\
+       https://api.github.com/repos/${STATUS_PROJECT}/statuses/${CI_COMMIT_SHA} \
+       -d "{\"state\":\"${CI_JOB_NAME}\",\"target_url\":\"${CI_PIPELINE_URL}\",\"context\":\"${STATUS_NAME}\"}"
+   # logic for when we find a way to get variables from the other jobs on different runners. Shared fiulesystem or mounting is the anwser.  
+    # echo "$jobid"
+     # echo "$WORKDIR/jobid"
+     # if [[ "$PARTITION_VARIABLE" == *"a100"*  ]]; then
+     #    gpu_arch= "a100-arch/80"
+     # fi
+
+     # if [[ "$PARTITION_VARIABLE" == *"dl"*  ]]; then
+     #    gpu_arch= "p100-arch/60"
+     # fi
+
+     # if [[ "$PARTITION_VARIABLE" == *"dlv"*  ]]; then
+     #    gpu_arch= "v100-arch/70"
+     # fi
+
+
+     # if [[ "$PARTITION_VARIABLE" == *"dlt"*  ]]; then
+     #    gpu_arch= "RTX2080-arch/75"
+     # fi
```

Comment:
This would just require running on an HPC-based runner - you would then have something in the pending stage save a file with the partition in the $WORKDIR, use that to post the status in this stage, and then run the after_script that I recommended.